github/testground.yml

```diff
@@ -158,7 +158,10 @@ repositories:
     visibility: public
   testground:
     branch_protection:
-      master: {}
+      master:
+        allows_force_pushes: false
+        required_pull_request_reviews:
+          required_approving_review_count: 1
```

Comment:
(manual edit to add branch protection)

Reply:
This doesn't have any effect at the moment because we're ignoring that property. Let's revert this commit, merge this PR and then start managing more branch protection rule props.